src/core/BufferAttribute.js

```diff
@@ -32,7 +32,17 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 }
 
-BufferAttribute.prototype = {
+Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {
+
+	set: function(value) { 
```

Comment:
Hmm, I guess you forgot to remove the `needsUpdate` setter from the `Object.assign()` down bellow. It seems to be compiling fine though?